src/utils/index.ts

```diff
@@ -28,3 +30,16 @@ export function getCellClassname<R, SR>(column: CalculatedColumn<R, SR>, ...extr
     ...extraClasses
   );
 }
+
+export function getColSpan<R, SR>(column: CalculatedColumn<R, SR>, lastFrozenColumnIndex: number, args: ColSpanArgs<R, SR>) {
+  const colSpan = typeof column.colSpan === 'function' ? column.colSpan(args) : column.colSpan;
```

Comment:
`column.colSpan` can't be a number anymore 🤔 
```suggestion
  const colSpan = typeof column.colSpan === 'function' ? column.colSpan(args) : 1;
```
```suggestion
  const colSpan = typeof column.colSpan === 'function' ? column.colSpan(args) : null;
```